tests/DevApps/aspnet-mvc/OwinWebApp/App_Start/Startup.Auth.cs

```diff
@@ -1,13 +1,11 @@
-﻿using Microsoft.Owin.Security;
-using Microsoft.Owin.Security.Cookies;
-using Owin;
-using Microsoft.Identity.Web;
-using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
-using Microsoft.Extensions.DependencyInjection;
+﻿using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Identity.Client;
-using Microsoft.Identity.Abstractions;
+using Microsoft.Identity.Web;
```

Comment:
Is this needed?

Reply:
In reading the PR description,  something doesn't seem right. Will need to look from laptop later

Reply:
This is just sorting using statements alphabetically